starters/deno-oak-denodb/src/graphql/resolvers/query_handler.ts

```diff
@@ -10,8 +11,7 @@ export const getTechnologies = async (
 	info: GraphQLResolveInfo,
 ): Promise<Technologies[]> => {
 	return await useCache<Technologies[]>(info.fieldName, cache, async () => {
-		const technologies = await Technologies.all();
-		return technologies;
+		return await TechnologyRepository.getAll();
```

Comment:
I'm just nitpicking but you don't need to await any of these, nor the useCache, nor the .getAll()

Reply:
Yeah, of course. Removed.